DNN Platform/Library/Entities/Portals/PortalController.cs

```diff
@@ -747,7 +1024,18 @@ public static string CheckDesktopModulesInstalled(XPathNavigator nav)
         /// <returns></returns>
         /// <remarks>
         /// </remarks>
-        public static string GetActivePortalLanguage(int portalID)
+        [Obsolete("Deprecated in DNN 9.7.0. Use the 'IPortalController.GetActivePortalLanguage' instance method. Scheduled removal in v11.0.0.")]
+        public static string GetActivePortalLanguage(int portalID) => Instance.GetActivePortalLanguage(portalID);
+
+        /// <summary>
+        ///   function provides the language for portalinfo requests
+        ///   in case where language has not been installed yet, will return the core install default of en-us.
+        /// </summary>
+        /// <param name = "portalID"></param>
+        /// <returns></returns>
+        /// <remarks>
+        /// </remarks>

```

Comment:
For the `IPortalController` implementations, let's just inherit docs from the interface:

```suggestion
        /// <inheritdoc />
```

Reply:
This makes sense, do we have this written down somewhere in the contributing guide?

Reply:
I'm going to mark this as resolved since the change has been implemented